packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -15,6 +13,10 @@ import { EngineReportingExtension } from './extension';
 import { GraphQLRequestContext, Logger } from 'apollo-server-types';
 import { InMemoryLRUCache } from 'apollo-server-caching';
 import { defaultEngineReportingSignature } from 'apollo-graphql';
+import {
+  Report,
+  TracesAndStats
+} from "apollo-engine-reporting-protobuf";
```

Comment:
Actually can you combine this with the existing import above? Look at the diff, you'll see what I mean.